.gitignore

```diff
@@ -12,6 +12,7 @@
 ehthumbs.db
 Thumbs.db
 .idea/*
+.python-version
```

Comment:
What does this ignore?

Reply:
I used a python version manager to handle different python versions and virtual environments, and it generates a .python-version file